src/components/TabsPage.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { Link, useParams } from 'react-router-dom';
+import { Tab } from '../react-app-env';
+
+type Props = {
+  tabs: Tab[],
+};
+
+const TabsPage: React.FC<Props> = ({ tabs }) => {
+  const { tabId } = useParams<{ tabId: string }>();
+  const findTab = tabs.find(tab => tabId === tab.id);
+
+  const content = tabId === undefined || findTab === undefined
+    ? 'Please select a tab'
+    : findTab.content;
+
+  return (
+    <section>
+      <nav className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => (
+            <li key={tab.id} className={tab.id === tabId ? 'is-active' : ''}>
```

Comment:
applying ternary with an empty string is a weak solution

Reply:
Mmm, how I can handle this case? Please give a hint!

Reply:
https://www.npmjs.com/package/classnames
:harold:

Reply:
It's only for practice? 
Because I think if I have **only** one class that I need to add , more obvious to use ternary operator. And don't download additional libraries for only custom one class.

Reply:
it's a common practice, you won't ever see such code samples in real projects 🤓 
my mission is to let you know such approaches like yours are inappropriate